docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Sites.md

```diff
@@ -0,0 +1,40 @@
+# Sites
+
+## Overview Study
+
+**Sites** or **centres** are terms used interchangeably in LORIS.
```

Comment:
wasn't there a consensus reached a few months ago about using the correct / superior 🇺🇸American 🗽 spelling of "centre"

Reply:
yes, let's stick with "center" per that meeting

Reply:
these emojis are making me wanna re-open that debate :canada:

Reply:
I picked `center`